scripts/run_multi_worker_app_paas.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+set -e -o pipefail
+
+TERMINATE_TIMEOUT=9
+
+function check_params {
+  if [ -z "${NOTIFY_APP_NAME}" ]; then
+    echo "You must set NOTIFY_APP_NAME"
+    exit 1
+  fi
+
+  if [ -z "${CW_APP_NAME}" ]; then
+    CW_APP_NAME=${NOTIFY_APP_NAME}
+  fi
+}
+
+function configure_aws_logs {
+  # create files so that aws logs agent doesn't complain
+  touch /home/vcap/logs/gunicorn_error.log
+  touch /home/vcap/logs/app.log.json
+
+  aws configure set plugins.cwlogs cwlogs
+
+  cat > /home/vcap/app/awslogs.conf << EOF
+[general]
+state_file = /home/vcap/logs/awslogs-state
+
+[/home/vcap/logs/app.log]
+file = /home/vcap/logs/app.log.json
+log_group_name = paas-${CW_APP_NAME}-application
+log_stream_name = {hostname}
+
+[/home/vcap/logs/gunicorn_error.log]
+file = /home/vcap/logs/gunicorn_error.log
+log_group_name = paas-${CW_APP_NAME}-gunicorn
+log_stream_name = {hostname}
+EOF
+}
+
+# For every PID, check if it's still running
+# if it is, send the sigterm
+function on_exit {
+  n=0
```

Comment:
Could use a better descriptor? wait_time? total_wait_time?

Reply:
I was just copying from the `run_app_paas.sh`, but ok, I'll change it in both.